README.md

```diff
@@ -1,76 +1,123 @@
+  files containing some JSON metadata about their format and origins.
+  Records directories are especially helpful for the ever-ambiguous
+  CSV format, where they solve the problem of 'hey, this may be a
+  CSV - but what's the schema?  What's the format of the CSV itself?
+  How is it escaped?'
 
-The record mover can be exended expand to handle additional database
-and data file types by building on top of their
-[SQLAlchemy](https://www.sqlalchemy.org/) drivers, and is able to
-auto-negotiate the most efficient way of moving data from one to the
-other.
+Records mover can be exended expand to handle additional databases
+and data file types.  Databases are supported by building on top of
+their [SQLAlchemy](https://www.sqlalchemy.org/) drivers.  Records
+mover is able to auto-negotiate the most efficient way of moving data
+from one to the other.
 
-Example CLI use:
+## Example CLI use
+
+Installing:
+
+```sh
+pip3 install 'records_mover[cli,postgres,redshift]'
+```
+
+Loading a CSV into a database:
+
+```sh
+mvrec file2table foo.csv redshiftdb1 myschema1 mytable1
+```
+
+Copying a table from a PostgreSQL to a Redshift database:
 
 ```sh
-pip3 install 'records_mover[movercli]'
 mvrec --help
-mvrec table2table mydb1 myschema1 mytable1 mydb2 myschema2 mytable2
+mvrec table2table postgresdb1 myschema1 mytable1 redshiftdb2 myschema2 mytable2
 ```
 
-For more installation notes, see [INSTALL.md](./INSTALL.md).  To
-understand the security model here, see [SECURITY.md](./SECURITY.md).
+Note records mover will automatically build an appropriate `CREATE
+TABLE` statement on the target end if the table doesn't already exist.
 
 Note that the connection details for the database names here must be
 configured using
 [db-facts](https://github.com/bluelabsio/db-facts/blob/master/CONFIGURATION.md).
 
-Example Python library use:
+For more installation notes, see [INSTALL.md](./INSTALL.md).  To
+understand the security model here, see [SECURITY.md](./SECURITY.md).
+
+## Example Python library use
 
 First, install records_mover.  We'll also use Pandas, so we'll install
-that, too:
+that, too, as well as a driver for Postgres.
 
 ```sh
-pip3 install records_mover pandas
+pip3 install records_mover[pandas,postgres]
 ```
 
 Now we can run this code:
 
 ```python
 #!/usr/bin/env python3
 
+# Pull in the records-mover library - be sure to run the pip install above first!
+from records_mover import sources, targets, move
+from pandas import DataFrame
+import sqlalchemy
+import os
+
+sqlalchemy_url = f"postgresql+psycopg2://username:{os.environ['DB_PASSWORD']}@hostname/database_name"
+db_engine = sqlalchemy.create_engine(sqlalchemy_url)
+
+df = DataFrame.from_dict([{'a': 1}]) # or make your own!
+
+source = sources.dataframe(df=df)
+target = targets.table(schema_name='myschema',
+                       table_name='mytable',
+                       db_engine=db_engine)
+results = move(source, target)
+```
```

Comment:
This is an example of using the simplified usage, where we're not using a Session object at all.